web-console/script/create-sql-docs.js

```diff
@@ -90,15 +90,15 @@ const readDoc = async () => {
 
   // Make sure there are enough functions found
   const numFunction = Object.keys(functionDocs).length;
-  if (numFunction < MINIMUM_EXPECTED_NUMBER_OF_FUNCTIONS) {
+  if (!(MINIMUM_EXPECTED_NUMBER_OF_FUNCTIONS <= numFunction)) {
```

Comment:
Is this some weird js quirk? How is this different from the original `numFunc < MIN_EXPECTED` condition?

Reply:
they behave differently if `numFunctions` is `undefined` because all comparisons with undefined are always false:

<img width="145" alt="image" src="https://user-images.githubusercontent.com/177816/189180402-14b543bd-35f2-4cef-aca0-04864660ed29.png">